config/initializers/authentication.rb

```diff
@@ -0,0 +1,21 @@
+# Check for authentication method and copy data over if necessary (ENV variable
+# to skip if necessary)
+unless ENV['SKIP_AUTH_INIT']
```

Comment:
This is what I was referring to in the comments; it allows us to optionally skip this intializer so that we can potentially run things like `rake db:reset` or really any rake command if `ENV['CAS_AUTH']` was set but your database didn't have `cas_login`s for all users.